twake/frontend/src/app/services/languages/locale/fr.js

```diff
@@ -36,14 +36,14 @@ export default {
   twake_not_on_device_browser:
     'Vous devez mettre à jour votre navigateur ou télécharger Twake pour ordinateur.',
   'ads.workspace_is_free_version':
-    "Besoin de plus de fonctionnalités ? Passez en version complète pour pouvoir ajouter plus de personnes, plus d'espaces de travail, espace de stockage illimité, etc.",
+    "Besoin de plus de fonctionnalités ? Passez en version complète pour pouvoir ajouter plus de utilisateurs, plus d'espaces de travail, espace de stockage illimité, etc.",
   'ads.activate_premium_to_add_workspaces':
     "Passez en version complète pour ajouter d'autres espaces de travail.",
   'ads.two_month_offer': 'Essayez Twake Premium gratuitement sans conditions pendant deux mois.',
   'ads.two_month_offer_confirmation':
     'Utiliser la version complète de Twake avec ce groupe gratuitement pendant deux mois.',
   'tutorial.start_to_add_workspaces':
-    'Commencez par ajouter un espace de travail partagé ou demander à une personne de vous inviter dans un espace existant.',
+    'Commencez par ajouter un espace de travail partagé ou demander à un utilisateur de vous inviter dans un espace existant.',
```

Comment:
@BenoitTallandier for this one maybe not great?

Reply:
Oui, on peut mettre un truc " ou demander à quelqu'un" :)

Reply:
Updated 👍🏻